optuna/study.py

```diff
@@ -709,6 +728,7 @@ def create_study(
     load_if_exists: bool = False,
     *,
     directions: Optional[Sequence[str]] = None,
+    heartbeat_interval: Optional[int] = None,
```

Comment:
Can/should we avoid introducing a storage dependent parameter to `create_study`? I'm wondering if we cannot fix it or make if configurable at the storage layer instead.